packages/@o3r/design/schematics/ng-add/register-generate-css/register-task.ts

```diff
@@ -8,17 +8,18 @@ import { posix } from 'node:path';
  * @param taskName name of the task to generate
  */
 export const registerGenerateCssBuilder = (projectName?: string, taskName = 'generate-css'): Rule => {
-  const registerBuilderRule: Rule = async (tree, {logger}) => {
+  const registerBuilderRule: Rule = async (tree, { logger }) => {
     const { getWorkspaceConfig, registerBuilder } = await import('@o3r/schematics');
-    const workspaceProject = projectName ? getWorkspaceConfig(tree)?.projects[projectName] : undefined;
-    const srcBasePath = workspaceProject?.sourceRoot || (workspaceProject?.root ? posix.resolve(workspaceProject.root, 'src') : '');
-    const themeFile = posix.resolve(srcBasePath, 'style', 'theme.scss');
+    const workspace = getWorkspaceConfig(tree);
+    const workspaceProject = projectName ? workspace?.projects[projectName] : undefined;
+    const srcBasePath = workspaceProject?.sourceRoot || (workspaceProject?.root ? posix.join(workspaceProject.root, 'src') : '.');
```

Comment:
Why is it '.' and not './src' ?

Reply:
indeed it could be to be more aligned with the case we have the project root